### DISCUSSION TITLE
Copy Option is not Available in Row Level Security

### DISCUSSION THREAD

**Discussion by vedfordev** [apache/superset#23806] [General]
I am unable to copy row level security. In Action it shows only delete option
<img width="302" alt="Screenshot 2023-04-25 at 11 50 07 AM" src="https://user-images.githubusercontent.com/106894932/234191099-3638ecf2-17a4-4e8f-960d-5fdc0e182a3f.png">

**Comment by LeoDiep** [comment]
+ 1
Please add this feature
